commons/addons/fs_buckets.md

```diff
@@ -154,12 +168,16 @@ buckets</td>
 
 ## Accessing your data inside the FS Bucket
 
+### From your application
+
+Your folder is mounted in application folder (`APP_HOME` env var). That mean `/some/empty/folder` is mounted in `/home/bas/app_01234567-0123-0123-0123-012345678987/some/empty/folder`.
```

Comment:
I'm not sure we should talk about the `APP_HOME` environment variable after all. Here and in the table above. That's not really something the user will use to mount buckets.

Also, what do you think of: `Your bucket is mounted at the configured path, starting from your application's root folder` or something like that ?

Reply:
Yes, your proposition
`Your bucket is mounted at the configured path, starting from your application's root folder`
is fine.